### ISSUE TITLE
I cannot build bigints with pulp/bower

### ISSUE THREAD

**Issue by newlandsvalley** [sharkdp/purescript-bigints#36]
My library uses bigints.  I can't build it with pulp/bower although I can with spago.  I wonder if it is related to the CI failure on the last commit?

**Comment by sharkdp** [comment]
Thank you for reporting this. I did not notice the CI failure. Maybe @thomashoneyman can help here? (see also: https://github.com/sharkdp/purescript-bigints/pull/33#issuecomment-790968162)

**Comment by thomashoneyman** [comment]
@newlandsvalley What error are you seeing, and on what Pulp / Bower versions? I just pulled the repository down locally and I was able to build using:

```sh
λ bower --version      
1.8.8

λ pulp --version
Pulp version 15.0.0
purs version 0.14.2
```

**Comment by thomashoneyman** [comment]
Is it possible you're seeing an error when running `npm run build` if you don't have `purescript-psa` installed on your system? I think it ought to work otherwise. I've opened #37 to update CI.

**Comment by newlandsvalley** [comment]
Currently I have : 

```
pulp --version
Pulp version 15.0.0
purs version 0.14.1

bower --version
1.8.12
````

I do ```pulp init``` and then edit ```bower.json``` to add ```"purescript-bigints": "^5.0.0"```.  If I then do ```bower install``` I see:

```
Unable to find a suitable version for purescript-effect, please choose one by typing one of the numbers below:
    1) purescript-effect#^2.0.0 which resolved to 2.0.1 and is required by purescript-refs#4.1.0, purescript-tailrec#4.1.1
    2) purescript-effect#^3.0.0 which resolved to 3.0.0 and is required by bigints, purescript-console#5.0.0, purescript-psci-support#5.0.0
````

I'm very confused about what's causing this.


```

**Comment by thomashoneyman** [comment]
It looks like your library may depend on `purescript-refs#4.1.0` or `purescript-tailrec#4.1.1`, but those are both older of those libraries (they're both on v5 now). Because those libraries are a version behind they're trying to require `effect#2.0.0`, but your other dependencies (console, bigints, etc.), which are on up-to-date versions, are trying to require `effect#3.0.0`.

It may be that you need to update your `refs` or `tailrec` dependencies, or perhaps you have some other dependency that's bringing those packages in.

**Comment by newlandsvalley** [comment]
But at this point, I have no library:

```
{
  "name": "mybigints",
  "ignore": [
    "**/.*",
    "node_modules",
    "bower_components",
    "output"
  ],
  "dependencies": {
    "purescript-prelude": "^5.0.1",
    "purescript-console": "^5.0.0",
    "purescript-effect": "^3.0.0",
    "purescript-bigints": "^5.0.0"
  },
  "devDependencies": {
    "purescript-psci-support": "^5.0.0"
  }
}
```

**Comment by thomashoneyman** [comment]
Huh! @sharkdp it looks like the v5.0.0 release was accidentally cut without including the PureScript 0.14 updates!

https://github.com/sharkdp/purescript-bigints/releases/tag/v5.0.0 points to https://github.com/sharkdp/purescript-bigints/commit/54863140def3f25d131b1a4994a4fa91ef51f017 but it should point to https://github.com/sharkdp/purescript-bigints/commit/a072ed2e81cf2b1f975e55c74fb7bd3a0c655c30.

**Comment by newlandsvalley** [comment]
Oh - thank goodness!  I thought I was going mad!

**Comment by thomashoneyman** [comment]
Me too 😂 once that's fixed you should be good to go. In the meantime you could point the Bower file at a specific commit instead of the v5.0.0 tag.

**Comment by newlandsvalley** [comment]
Yeah - there's no rush.  I think I'll wait for the new version (which hopefully might now be published to pursuit).  Many thanks for following this up so quickly @thomashoneyman and @sharkdp.

**Comment by sharkdp** [comment]
> Huh! @sharkdp it looks like the v5.0.0 release was accidentally cut without including the PureScript 0.14 updates!

Looks like I forgot to `pull` before tagging. Sorry for that. I released v6.0.0 now.
